src/global.d.ts

```diff
@@ -1,22 +0,0 @@
-// The ResizeObserver API is available in all major browsers. However,
-// TypeScript has yet to implement any type declarations for it. A basic
-// incomplete version of the API is declared here just to get the compiler
-// not to throw errors.
-//
-// https://github.com/microsoft/TypeScript/issues/37861
-
-interface IResizeObserver {
```

Comment:
What am I doing wrong here?

![image](https://user-images.githubusercontent.com/154725/119104430-89e0e500-ba1c-11eb-85ec-2067ba78b798.png)

I've run `npm install`, and `npm list typescript` yields:

```
@limetech/lime-elements@31.9.0 /Users/adrian.schmidt/src/lime-elements
└── typescript@4.2.3
```

Reply:
Hmm :thinking: 
It seems a bit odd... I get the same output
```
@limetech/lime-elements@31.8.0 /home/jth/workspace/lime-elements
└── typescript@4.2.3
```

Maybe another case of Mac vs Linux? :disappointed:

Reply:
I've updated Stencil to 2.6.0 in #1274, and with your fix cherry-picked, that passes both on GitHub and locally on my machine, so I'll merge that one, and close this.